google-cloud-datastore/lib/google/cloud/datastore/service.rb

```diff
@@ -144,6 +153,14 @@ def generate_read_options consistency, transaction
           nil
         end
 
+        def default_headers
+          { "google-cloud-resource-prefix" => "projects/#{@project}" }
```

Comment:
Sorry for my lack of ruby-foo -- just to double check that for project "my-project" this turns into: "projects/my-project"?